static/app/utils/discover/fields.tsx

```diff
@@ -79,6 +79,11 @@ export type QueryFieldValue =
       kind: 'field';
       alias?: string;
     }
+  | {
+      field: string;
+      kind: 'calculatedField';
```

Comment:
nit: maybe not within scope of this pr either but I think we can probably also use the `FieldValueKind` enum here